### PR TITLE
fix intervals checks

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by maxulysse** [comment]
Got a warning about intervals while running --step annotate

File: conf/test.config [hunk 1]
```diff
@@ -54,6 +54,7 @@ profiles {
         params.step                = 'annotate'
     }
     no_intervals {
+        params.intervals           = null
```
Comment (FriederikeHanssen) [inline conf/test.config:57]: do we need to add the same to the nextflow config/ some logic for params.no_intervals?

Comment (maxulysse) [reply conf/test.config:57]: I'm assuming that people specifying `--no_intervals` mostly do that with a custom genome, and don't provide `--intervals` then.
Worst case for people using `--no_intervals` and iGenomes for genomes where we do provide `intervals`, we have just this warning.

**Review by asp8200 — APPROVED** [review]
Looks fine to me.

File: conf/test.config (hunk 1 above)
Comment (FriederikeHanssen) [reply conf/test.config:57]: ok

**Pull request merged**